$NetBSD: patch-clutter-gtk_Makefile.am,v 1.1 2014/01/04 13:34:16 obache Exp $

* using deprecated API
* build: Add common CFLAGS to the introspection scanner
  https://git.gnome.org/browse/clutter-gtk/commit/?h=clutter-gtk-0.10&id=4de105a2bbd201c8e60f3e8777794a4cd532afdc

--- clutter-gtk/Makefile.am.orig	2010-09-16 13:44:00.000000000 +0000
+++ clutter-gtk/Makefile.am
@@ -7,7 +7,6 @@ AM_CPPFLAGS = \
 	-DPREFIX=\""$(prefix)"\" 		\
 	-DLIBDIR=\""$(libdir)"\" 		\
         -DCLUTTER_GTK_COMPILATION               \
-        -DG_DISABLE_DEPRECATED                  \
         -DGDK_PIXBUF_DISABLE_DEPRECATED         \
         -DGDK_DISABLE_DEPRECATED                \
         -DGTK_DISABLE_DEPRECATED                \
@@ -15,6 +14,8 @@ AM_CPPFLAGS = \
 	-DCLUTTER_DISABLE_DEPRECATED		\
 	$(CLUTTER_GTK_DEBUG_CFLAGS)
 
+common_cflags = $(CLUTTER_CFLAGS) $(GTK_CFLAGS)
+
 # please, keep these lists sorted alphabetically
 sources_c = \
 	$(srcdir)/gtk-clutter-embed.c 		\
@@ -36,7 +37,7 @@ lib_LTLIBRARIES = libclutter-gtk-@CLUTTE
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_SOURCES = $(sources_c)
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LIBADD = $(CLUTTER_LIBS) $(GTK_LIBS)
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LDFLAGS = $(CLUTTER_LT_LDFLAGS)
-libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_CFLAGS = $(MAINTAINER_CFLAGS) $(GTK_CFLAGS) $(CLUTTER_CFLAGS) 
+libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_CFLAGS = $(common_cflags) $(MAINTAINER_CFLAGS)
 
 cluttergtkheadersdir = $(includedir)/clutter-1.0/clutter-gtk
 cluttergtkheaders_HEADERS = $(sources_h) $(srcdir)/clutter-gtk.h
@@ -48,7 +49,7 @@ GtkClutter-@CLUTTER_GTK_API_VERSION@.gir
 	$(AM_V_GEN)$(INTROSPECTION_SCANNER) -v \
 		--namespace GtkClutter --nsversion=@CLUTTER_GTK_API_VERSION@ \
 		--strip-prefix=GtkClutter \
-		$(INCLUDES) $(AM_CPPFLAGS) \
+		$(INCLUDES) $(AM_CPPFLAGS) $(common_cflags) \
 		--add-include-path=$(srcdir) --add-include=path=. \
 		--include=Clutter-1.0 \
 		--include=GdkPixbuf-2.0 \
